streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/master/JobRuntimeContext.java

```diff
@@ -0,0 +1,46 @@
+package org.ray.streaming.runtime.master;
+
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import org.ray.streaming.jobgraph.JobGraph;
+import org.ray.streaming.runtime.config.StreamingConfig;
+import org.ray.streaming.runtime.core.graph.Graphs;
+import org.ray.streaming.runtime.core.graph.executiongraph.ExecutionGraph;
+
+/**
+ * Runtime context for job master.
+ * Including: graph, resource, checkpoint info, etc.
+ */
+public class JobRuntimeContext implements Serializable {
+
+  private StreamingConfig conf;
+  private volatile Graphs graphs;
```

Comment:
I don't think we need `Graphs` class, put all it's members to here is enough

Reply:
Remove ```Graphs```.